packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -186,7 +186,7 @@ const getProps = ({
       isVisited,
     }) as IconProps['color'],
     fontSize: size === 'medium' ? 100 : 75,
-    lineHeight: size === 'medium' ? 'm' : 's',
+    lineHeight: size === 'medium' ? 'l' : 's',
```

Comment:
This was an incorrect implementation 🙈. It is `l` instead of `m` on Figma as well.